src/main/content/_assets/css/guide.scss

```diff
@@ -53,10 +53,7 @@ header {
 }
 
 #guide-content h3 {
-    font-size: 16px;
```

Comment:
if this is removed it will default to 20px font-size which would be the same as the h2 heading.

Maybe we can update to use rem for font-size here:

h1 -> 2rem
h2 -> 1.7rem
h3 -> 1.4rem
h4 -> 1.1rem

and also update h3 to have font-weight of 600. That should do the trick

Reply:
![image](https://user-images.githubusercontent.com/3623618/68137958-c958e680-fef5-11e9-8c1f-f04b5f48f852.png)

Reply:
I've updated the code to use some relative sizes 😊